frontend/src/components/desktop_apps/screens/TitleBar.tsx

```diff
@@ -3,6 +3,7 @@ import { AppType } from "../../../../../shared/types";
 import { useKeyDown } from "../../../../frontend-util";
 import { IGameState } from "../../../store";
 import { playSoundEffect } from "../../../soundEffect";
+import { setPreviousCooldown } from "../../../slices/gameDataSlice";
```

Comment:
Is this an unused import? Also add a newline between the last import and the interface thx.